imports/plugins/included/notifications/client/containers/notification.js

```diff
@@ -0,0 +1,38 @@
+import { compose, withProps } from "recompose";
```

Comment:
Generally I don't know if I am favor of renaming these. I found having `container` in the name makes it easier to find the appropriate file, and no one died from being too explicit.